docs/code-quality/ca1801.md

```diff
@@ -35,6 +35,10 @@ This rule does not examine the following kinds of methods:
 
 - Methods used as event handlers.
 
+- Serialization constructors ([see guidelines](https://docs.microsoft.com/dotnet/standard/serialization/serialization-guidelines#supporting-runtime-serialization)).
```

Comment:
Is it ok to use this syntax?

Reply:
It's OK to use. Best syntax for a docs link like this is: /dotnet/standard/serialization/serialization-guidelines#supporting-runtime-serialization